packages/flutter/test/material/choice_chip_test.dart

```diff
@@ -132,4 +132,52 @@ void main() {
     final RawChip rawChip = tester.widget(find.byType(RawChip));
     expect(rawChip.iconTheme, iconTheme);
   });
+
+  testWidgets('ChoiceChip passes showCheckMark property to RawChip', (WidgetTester tester) async {
+    const bool showCheckmark = false;
+    await tester.pumpWidget(wrapForChip(
+        child: const ChoiceChip(
+          label: Text('Test'),
+          selected: true,
+          showCheckmark: showCheckmark,
+        )));
+    final RawChip rawChip = tester.widget(find.byType(RawChip));
+    expect(rawChip.showCheckmark, showCheckmark);
+  });
+
+  testWidgets('M3 width should not change with selection', (WidgetTester tester) async {
```

Comment:
This doesn't look like it belongs in this PR. 
The PR's for `showCheckmark` isn't it?

Reply:
I referred to this test case in the `FilterChip` widget tests. You're right, this unit test is not directly related to this PR. I'll remove it.